### DISCUSSION TITLE
Keycloak cookie issue

### DISCUSSION THREAD

**Discussion by Swati1625** [keycloak/keycloak#20479] [Q&A]
When the first-time login with Key cloak, requests are going to Key cloak server through api  the following cookies are generated to manage the user
information.

1.AspNet.CookiesC1 (Cookie Size is 4047 Bytes)
2.AspNet.CookiesC2(Cookie Size is 281 Bytes)

this is the actual scenario, so it’s working fine.

When the second-time login with the same browser (another tab), again, requests are going to Key cloak
server through api the following cookies are
generated to manage the user information.

1.AspNet.ExternalCookies.C1(Cookie Size is 4047 Bytes)
2. AspNet.ExternalCookies.C2(Cookie Size is 401 Bytes)

Bad request error come after getting generated the second set of cookies, because **cookie sizes are
reaching than the actual browser limit**

**Comment by slominskir** [comment]
See: https://github.com/keycloak/keycloak/issues/16181

**Reply by Swati1625** [reply]
@slominskir I can not see two cookies with the same value or different SameSite
